services/venice-server/src/main/java/com/linkedin/venice/listener/ReadQuotaEnforcementHandler.java

```diff
@@ -465,6 +473,10 @@ private void removeTopics(Set<String> topicsToRemove) {
     for (String topic: topicsToRemove) {
       customizedViewRepository.unSubscribeRoutingDataChange(topic, this);
       storeVersionRateLimiters.remove(topic);
+      ServerReadQuotaUsageStats storeStats = stats.getStoreStats(Version.parseStoreFromKafkaTopicName(topic));
+      if (storeStats != null) {
```

Comment:
`getStoreStats` will always return a non-null (does `computeIfAbsent`). You will need a new API to check contains.

Reply:
Ops, good catch. Done.